WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorViewModel.swift

```diff
@@ -175,3 +197,20 @@ extension NewTaxRateSelectorViewModel {
         syncState = taxRateViewModels.isNotEmpty ? .results: .empty
     }
 }
+
+extension NewTaxRateSelectorViewModel {
+    enum Localization {
+        static let bottomNoticeResultsSectionTitle = NSLocalizedString("Can’t find the rate you’re looking for?",
+                                                                         comment: "Text to prompt the user to edit tax rates in the web")
+        static let bottomNoticeEmptySectionTitle = NSLocalizedString("You don't have any tax rate with a location.",
```

Comment:
```suggestion
        static let bottomNoticeEmptySectionTitle = NSLocalizedString("You don't have any tax rates with a location.",
```

Reply:
It's great to have a native speaker in the team, thanks! Changed in 787d8789ef48f7ee509eae4a70a22c95a507ef0f